custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -115,7 +119,14 @@ function MonthFilterController($scope, $location, $uibModal, storageService) {
             controllerAs: '$ctrl',
             resolve: {
             },
-        });
+        };
+
+        if (vm.isSSD && !vm.isSSDAvailableForGivenDate) {
+            modalOptions['backdrop'] = 'static';
```

Comment:
what does this do?

Reply:
Generally if we do a escape for click outside of the model it will dismiss the model. In this condition we are making the apply button necessary by removing those.